### ISSUE TITLE
Suggestion: Add help for gecko driver

### ISSUE THREAD

**Issue by crantok** [tebeka/selenium#38]
As per instructions in doc.go, I downloaded the selenium standalone server. I obtained it from a different URL from the one given in doc.go (see my pull request.) The server was major version 3. I got the following error when calling selenium.NewRemote()

    The path to the driver executable must be set by the webdriver.gecko.driver system property;
    for more information, see https://github.com/mozilla/geckodriver.
    The latest version can be downloaded from https://github.com/mozilla/geckodriver/releases

To solve this, I downloaded a precompiled driver from

    https://github.com/mozilla/geckodriver/releases

as instructed and added the path to that driver in my selenium.Capabilities instance

```go
	caps := selenium.Capabilities{
		"browserName":            "firefox",
		"webdriver.gecko.driver": "/path/to/downloaded/geckodriver",
	}
```
The example in doc.go worked after that.

**Comment by tebeka** [comment]
Thanks! I'll add to help soon-ish.

**Comment by crantok** [comment]
If I get time today, I'll edit doc.go myself and submit a pull request.

**Comment by tebeka** [comment]
Closed in PR #39
